app/main.py

```diff
@@ -1 +1,26 @@
-# write your code here
+import os
+
+
+def copy_file(my_file: str) -> None:
+    parts = my_file.split()
```

Comment:
You could follow another approach by avoiding value retrieval by indexing. In this way code would be more descriptive 
```suggestion
    command, source_file, destination_file, *_ = my_file.split()
```
Follow this approach in the future